geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/AnalyticFeature.java

```diff
@@ -37,7 +37,10 @@ public static SimpleFeature createGeometryFeature(
 			final int zoomLevel,
 			final int iteration,
 			final long count ) {
-		assert (extraDimensionNames.length == extraDimensions.length);
+		if (extraDimensionNames.length != extraDimensions.length) {
```

Comment:
Style question here - 
my take is (since assert() is disabled at runtime unless you change default JVM params) we really don't want assert statements.    If they are used for argument validation then they should be moved to a pattern like this, with a useful message.  (or just removed)

That gets into what we want to set as style guidelines for the project though.